client/src/modules/invoices/registry/modalCreditNote.js

```diff
@@ -3,22 +3,33 @@ angular.module('bhima.controllers')
 
 ModalCreditNoteController.$inject = [
   '$uibModalInstance', 'PatientInvoiceService', 'data', 'VoucherService', 'NotifyService',
-  '$translate',
+  '$translate', 'SessionService'
 ];
 
-function ModalCreditNoteController(Instance, Invoices, data, Vouchers, Notify, $translate) {
+function ModalCreditNoteController(Instance, Invoices, data, Vouchers, Notify, $translate, SessionService) {
   var vm = this;
 
   vm.creditNote = {};
   vm.submit = submit;
   vm.cancel = function cancel() { Instance.close(false); };
+  vm.currencySymbol = SessionService.enterprise.currencySymbol;
 
   vm.creditNote.uuid = data.invoice.uuid;
   vm.patientInvoice = data.invoice;
+  vm.billingAmount = 0;
+  vm.subsidyAmount = 0;
 
   Invoices.read(data.invoice.uuid)
     .then(function (response) {
       vm.patientInvoiceItems = response.items;
+     
+      response.billing.forEach(bill => {
```

Comment:
Unfortunately, we can't use arrow functions on the client as it breaks our `gulp` build in production.  Could you make these into `function ()` calls?